tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts

```diff
@@ -364,10 +411,13 @@ export class LineChartComponent
   }
 
   /**
-   * Minimally and imperatively updates the chart library depending on prop changed.
+   * Minimally and imperatively updates the chart library depending on prop
+   * changed. When adding new `lineChart.*()` calls, keep this in sync with
```

Comment:
I do not see any `lineChart.*` API invoked in `recoverRendererIfNeeded`. Do you truly need them to be synced?

Reply:
Updated the comment to be `this.lineChart.set*()`.  If we added a `this.lineChart.setZMode(...)` inside `updateLineChart` one day, we'd need the corresponding flag `this.zModeUpdated = true` inside `recoverRendererIfNeeded` so that any changes to initial values of this hypothetical "zMode" would persist after recovery.